src/app/schools/schools.service.ts

```diff
@@ -0,0 +1,15 @@
+import {Injectable} from '@angular/core';
+import {School} from './school';
+import {EntityMapperService} from '../entity/entity-mapper.service';
+import {Observable} from 'rxjs';
+
+@Injectable()
+export class SchoolsService {
+
+  constructor(private entityMapper: EntityMapperService) {
+  }
+
+  getSchools(): Observable<School[]> {
+    return Observable.fromPromise(this.entityMapper.loadType<School>(School));
```

Comment:
We are currently updating to Angular 7, where this is no longer allowed. We should instead use

```ts
import {from} from 'rxjs';
//...
return from(this.entityMapper.loadType<School>(School));
```

We changed this everywhere else in our branch already